pulp_file/tests/functional/api/test_download_policies.py

```diff
@@ -139,7 +139,11 @@ def test_download_policy(
     actual_checksum = hashlib.sha256(downloaded_file.body).hexdigest()
     expected_checksum = content_unit[1]
     assert expected_checksum == actual_checksum
-    if download_policy == "immediate" and settings.DEFAULT_FILE_STORAGE in OBJECT_STORAGES:
+    if (
+        download_policy == "immediate"
+        and settings.DEFAULT_FILE_STORAGE in OBJECT_STORAGES
```

Comment:
From the logic of the settings, I think this should be
```suggestion
        and settings.DEFAULT_FILE_STORAGE != "pulpcore.app.models.storage.FileSystem"
```